templates/_schemas/definitions/person.twig

```diff
@@ -0,0 +1,11 @@
+{% macro person(props, schema = null) %}
+  {% from '_schemas/definitions/thing' import thing %}
+  
+  {% set schema = schema ?? craft.schema.Person %}
+  {% set schema = thing(props, schema) %}
+  {% set schema = schema
+    .name(props.name)
```

Comment:
Won’t `name` be set by `Thing`? Why do we need to repeat it here?

Reply:
In the `Thing` definition it is `props.title` and I think for the `Person` definition it would be better to use `props.name`.

Reply:
In this pull request https://github.com/miranj/craft-boilerplate/pull/55/files the first preference for the `name` property is given to `props.name` with a fallback to `props.title`.

Reply:
Oh, okay. Updated my development environment with a merge from `dev` and removed repeated property.